src/index.js

```diff
@@ -17,6 +17,28 @@ export function t(strings, ...exprs) {
     return strings;
 }
 
+const separator = /(\${\s*\d+\s*})/g;
+const slotIdRegexp = /\${\s*(\d+)\s*}/;
+
+export function jt(strings, ...exprs) {
+    if (strings && strings.reduce) {
+        const id = getMsgid(strings, exprs);
+        const transObj = findTransObj(currentLocale, id);
+        if (!transObj) return [id];
+
+        // splits string & capturing group into tokens
+        //
+        const translatedTokens = transObj.msgstr[0].split(separator);
```

Comment:
Maybe it's better to implement tockens match in this way:
```js
// without capturing group
const separator = /\${\s*\d+\s*}/g;
const translatedTokens = transObj.msgstr[0].split(separator);
const result = [];
translatedTockens.forEach((tocken, i) => {
    result.push(tocken);
    if (exprs[i] !== undefined) {
         result.push(exprs[i]);
    }
});
```
Benefits:
1. Minus one regex match.
2. No str -> int coercion `+slotIdMatch[1]`
What do you think?